tests/testthat/test-object-package.R

```diff
@@ -0,0 +1,17 @@
+context("Object: Package")
+
+test_that("author with more than one given/family name ", {
+  expect_equal(author_desc(
+                 person(
```

Comment:
This should just be indented by four spaces - otherwise looks great!

Reply:
Sorry. I used `styler` now.